libkbfs/constants.go

```diff
@@ -15,3 +15,7 @@ const rekeyRecheckInterval = 30 * time.Second
 
 // rekeyInitialTTL is the maximum number rechecks each rekey request can trigger.
 const rekeyInitialTTL = 4
+
+// reconnectBackoffWindow is a backoff window within which we try to wait
+// randomly for before reconnecting to MD server.
+const reconnectBackoffWindow = 2 * time.Minute
```

Comment:
The comment says this is for the MD server -- should it be in `mdserver_remote.go` then?  Or should `MD` be in the name somewhere?